home/migrations/0006_auto_20200623_2207.py

```diff
@@ -28,14 +28,15 @@ def migrate_appusers(apps, schema_editor):
             ON LOWER(au.email)=LOWER(ac.email)
         """)
         # create some dummy rows to set initial default values in migration
-        cursor.execute("""
-            INSERT INTO home_account (id, email, name, zip, age, created, updated)
-            VALUES (0, 'dummy@email.com', 'Dummy Account', '94103', 0, NOW(), NOW())
-        """)
-        cursor.execute("""
-            INSERT INTO home_device (device_id, account_id, created)
-            VALUES ('0', 0, NOW())
-        """)
+        # (2021-11-03/lee51) Commenting out the following (as unnecessary)
```

Comment:
Normally we don't modify migrations ex post facto, but I think the purpose of the following had been to validate the migration itself up to this point, so I don't think it's really necessary. Beyond that, it's actually a bit of a nuisance that doesn't seem to serve any useful purpose for development.

I believe the change is safe - it won't affect staging or prod unless we roll back to previous migration (which we should avoid anyhow since it could be destructive). The effect of reverting this migration would result in a `noop` for `migrate_appusers` so it would not try to remove row 0. Therefore, the only effect I can foresee is that new deployments (i.e. dev) will no longer have a dummy account row upon initialization.